src/main/java/helper/Constant.java

```diff
@@ -11,7 +11,7 @@
     public static final int TIMES_WAIT_ELEMENTS = 120;
     public static final int LONG_TIME_WAIT = 30;
     public static final String DATE_FORMAT = "M/d/yyyy";
-    public static final String FAIL_MSG_LOGIN_MULTIPLE = "You have used 4 out of 5 login attempts. After all 5 have been used, you will be unable to login for 15 minutes.";
+    public static final String INVALID_LOGIN_MULTIPLE_MSG = "You have used 4 out of 5 login attempts. After all 5 have been used, you will be unable to login for 15 minutes.";
```

Comment:
should be: 
MAXIUM_LOGIN_ATTEMPTS_WARNING_MSG

Reply:
Thanks for your review. I just have fixed and pushed into this branch. Please review my newest update.